firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -62,4 +57,18 @@
             .build(),
         LibraryVersionComponent.create("fire-perf", BuildConfig.VERSION_NAME));
   }
+
+  public FirebasePerformance providesFirebasePerformance(ComponentContainer container) {
```

Comment:
Should this be static? (If yes, please update line 51 to not use "this")

Reply:
Yes, it should be and private as well.